gnssanalysis/gn_io/trop.py

```diff
@@ -9,41 +9,70 @@
 from .. import gn_io as _gn_io
 
 
-def _read_tro_solution(path: str, recenter: bool = True) -> _pd.DataFrame:
+def _read_tro_solution(path: str, recenter: bool = True, trop_mode="Ginan") -> _pd.DataFrame:
+    """For backwards compatibility"""
+    return read_tro_solution(path, recenter=recenter, trop_mode=trop_mode)
+
+
+def read_tro_solution(path: str, recenter: bool = True, trop_mode="Ginan") -> _pd.DataFrame:
     """Parses tro snx file into a dataframe.
```

Comment:
One-line sphinx docstring (missing the description of input parameters, errors raise, what's returned, etc.: `:param` `:return`)